<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ScanWidgetBase</class>
 <widget class="QWidget" name="ScanWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Current Folder :</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="folder" />
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="imageList" >
       <property name="iconSize" >
        <size>
         <width>140</width>
         <height>180</height>
        </size>
       </property>
       <property name="isWrapping" stdset="0" >
        <bool>true</bool>
       </property>
       <property name="resizeMode" >
        <enum>QListView::Adjust</enum>
       </property>
       <property name="spacing" >
        <number>12</number>
       </property>
       <property name="viewMode" >
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="refresh" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>&amp;Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="informations" >
       <property name="title" >
        <string>File informations</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Number of pages :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="info_nbPages" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Scanned resolution :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="info_dpi" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Image size :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="info_resolution" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="options" >
       <property name="title" >
        <string>Options</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="6" column="1" >
         <widget class="QPushButton" name="delete" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>&amp;Delete</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QComboBox" name="color" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Color</string>
          </property>
          <property name="buddy" >
           <cstring>color</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="page" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Page</string>
          </property>
          <property name="buddy" >
           <cstring>page</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QComboBox" name="resolution" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Resolution</string>
          </property>
          <property name="buddy" >
           <cstring>resolution</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="format" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <item>
           <property name="text" >
            <string>tiff</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>gif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>jpeg</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>bmp</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>pdf</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Format</string>
          </property>
          <property name="buddy" >
           <cstring>format</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QPushButton" name="save" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>&amp;Save</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>93</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
